packages/system/kamaji/Makefile

```diff
@@ -1,18 +1,10 @@
 NAME=kamaji
-NAMESPACE=cozy-kamaji
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	flux suspend hr kamaji -n cozy-kamaji || true
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
 	helm repo add clastix https://clastix.github.io/charts
 	helm repo update clastix
-	helm pull clastix/kamaji --untar --untardir charts
+	helm pull clastix/$(NAME) --untar --untardir charts
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.